### ISSUE TITLE
[FIXED] Make fullscreen background black

### ISSUE THREAD

**Issue by sergeevabc** [omarroth/invidious#475]
Dear Omar,
to replace too bright white background of Firefox 63 (figure 1),
which is present when any site is not yet loaded,
I added a few settings as follows:
```
$ cat userChrome.css
 
@namespace url("http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"); 
.browserContainer { background-color: DimGrey !important; }

$ cat userContent.css

@namespace url(http://www.w3.org/1999/xhtml);
@-moz-document url("about:blank"), url("about:newtab"), url("about:home") {  
    body {
        background-color: DimGrey;
    }
}
```
It works as expected, eyes are happy (figure 2). Now when I come to any videopage on Invidious and switch playback to fullscreen mode by means of pressing F or clicking player’s right corner button, the background behind the video is not black, but gray (figure 3). It is very annoying, because players on other sites (Youtube, Hooktube, Cadence’s Cloudtube, Vimeo, etc) display solid black. Could you fix it, please?

![1]

![2]

![3]

[1]: https://i1.imageban.ru/out/2019/04/09/bc22644420439078d83e8e586fedaa3b.png
[2]: https://i4.imageban.ru/out/2019/04/09/6a60a4af080194a4f25f3a22b4379cda.png
[3]: https://i1.imageban.ru/out/2019/04/09/ee0934f4948567a7209d80fcdec3e6d8.png

**Comment by omarroth** [comment]
I'm having trouble reproducing this in Firefox 66 using the `userContent.css` and `userChrome.css` you provided. The issue may be with the styling you have applied, although I'm not sure why it would only apply to Invidious. I'm also confused by the gray border along the bottom. Attempting to reproduce I see this:
![image](https://user-images.githubusercontent.com/14208607/55926910-c3f22280-5bd8-11e9-91bc-1993b08bb52a.png)

With no gray border.

**Comment by sergeevabc** [comment]
@omarroth, while copying contents of `userContent.css` its little part with `url("about:blank")` got trimmed somehow. Please, try again.

```
$ cat userContent.css

@namespace url(http://www.w3.org/1999/xhtml);
@-moz-document url("about:blank"), url("about:newtab"), url("about:home") {  
    body {
        background-color: DimGrey;
    }
}
```

**Comment by omarroth** [comment]
Using the updated `userContent.css` I'm able to reproduce. Should be fixed with 5de300fb359af6d7428e6307e937b4d25e8f36b2.

**Comment by sergeevabc** [comment]
@omarroth, thank you for a quick reply, it works on invidio.us!
